### PR TITLE
chore(deps): update all to latest version

### PR DESCRIPTION
**Pull request by maninak**
Unfortunately still no fix for this

https://github.com/nuxt-modules/icon/issues/46#issuecomment-1406559857

that I hoped for and we need to re-enable typechecking on CI.

### CODE REVIEW

**Review by QZera — COMMENTED** [review]
After merging with `main` locally, I get this error on `pnpm dev`.
Everything seems to run just fine though.
Is this something to be concerned about?

```
 ERROR  [worker reload] Cannot read properties of null (reading 'address')                                                                                                                                                                      7:31:34 PM

  at _reload (node_modules/.pnpm/nitropack@2.8.0/node_modules/nitropack/dist/shared/nitro.37241ef0.mjs:282:38)
  at runNextTicks (node:internal/process/task_queues:60:5)
  at process.processTimers (node:internal/timers:511:9)
  at async _applyPromised (node_modules/.pnpm/perfect-debounce@1.0.0/node_modules/perfect-debounce/dist/index.mjs:54:10)
  ```

**Comment by maninak** [comment]
> After merging with `main` locally, I get this error on `pnpm dev`. Everything seems to run just fine though. Is this something to be concerned about?
> 
> ```
>  ERROR  [worker reload] Cannot read properties of null (reading 'address')                                                                                                                                                                      7:31:34 PM
> 
>   at _reload (node_modules/.pnpm/nitropack@2.8.0/node_modules/nitropack/dist/shared/nitro.37241ef0.mjs:282:38)
>   at runNextTicks (node:internal/process/task_queues:60:5)
>   at process.processTimers (node:internal/timers:511:9)
>   at async _applyPromised (node_modules/.pnpm/perfect-debounce@1.0.0/node_modules/perfect-debounce/dist/index.mjs:54:10)
> ```

Good catch. [According to the maintainer](https://github.com/nuxt/nuxt/issues/24423#issuecomment-1824258988) it seems to be safe to ignore and should be resolved soon.

**Review by QZera — APPROVED** [review]
LGTM

**Pull request merged**
